plugins/topology/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@janus-idp/backstage-plugin-topology",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
```

Comment:
```suggestion
```

Please make the package public so we can release it.

Reply:
Thanks Tom, updated